README.md

```diff
@@ -78,9 +79,17 @@ func main() {
 }
 ```
 
-### 4. Create your model registry
 
-Create `models/models_registry.go` in your project:
+### 4. Generate your model registry
+
+Use the register command to automatically scan your models directory (e.g., models/) and generate a models_registry.go file.
+
+
+```bash
+go run cmd/migration/main.go register 
```

Comment:
Can you update the command here to include the path to the models? i.e 

`go run cmd/migration/main.go register [path/to/models]`